docs/src/main/asciidoc/spring-security-support.adoc

```diff
@@ -25,26 +25,30 @@ The `spring-cloud-azure-starter-active-directory` (`aad-starter` for short) prov
-|*spring.cloud.azure.active-directory*.authorization-clients.AZURE_CLIENT_NAME.on-demand |This is used for incremental consent. The default value is false. If it's true, it's not consent when user login, when application needs the additional permission, incremental consent is performed with one OAuth2 authorization code flow.
-|*spring.cloud.azure.active-directory*.authorization-clients.AZURE_CLIENT_NAME.scopes |API permissions of a resource server that the application is going to acquire.
-|*spring.cloud.azure.active-directory*.credential.client-id |Registered application ID in Azure AD.
-|*spring.cloud.azure.active-directory*.credential.client-secret |Client secret of the registered application.
-|*spring.cloud.azure.active-directory*.post-logout-redirect-uri |Redirect uri for posting log-out.
-|*spring.cloud.azure.active-directory*.profile.environment.active-directory-endpoint |Base uri for authorization server, the default value is `https://login.microsoftonline.com/`.
-|*spring.cloud.azure.active-directory*.profile.environment.microsoft-graph-endpoint |Microsoft graph endpoint. The default value is `https://graph.microsoft.com/`.
-|*spring.cloud.azure.active-directory*.profile.tenant-id |Azure Tenant ID.
-|*spring.cloud.azure.active-directory*.redirect-uri-template |Used by the authorization server to return responses containing authorization credentials to the client via the resource owner user-agent.
-|*spring.cloud.azure.active-directory*.resource-server.claim-to-authority-prefix-map |Claim to authority prefix map. Default map is: "scp" -&gt; "SCOPE_", "roles" -&gt; `APPROLE_`.
-|*spring.cloud.azure.active-directory*.resource-server.principal-claim-name |Principal claim name. Default value is "sub".
-|*spring.cloud.azure.active-directory*.user-group.allowed-group-ids |Users' group id can be use in `@PreAuthorize(&quot;hasRole(&#39;ROLE_group_id_1&#39;)&quot;)`. Not all group id will take effect, only group id configured in this property will take effect. If this property's value is `all`, then all group id will take effect.
-|*spring.cloud.azure.active-directory*.user-group.allowed-group-names |Users' group name can be use in `@PreAuthorize(&quot;hasRole(&#39;ROLE_group_name_1&#39;)&quot;)`. Not all group name will take effect, only group names configured in this property will take effect.
-|*spring.cloud.azure.active-directory*.user-group.use-transitive-members |Whether to use `/v1.0/me/transitiveMemberOf` instead of  `/v1.0/me/memberOf` to get groups.
-|*spring.cloud.azure.active-directory*.user-name-attribute |Decide which claim to be principal's name. 
+|Name |Default |Description
```

Comment:
The two tables are not rendering correctly 
![image](https://user-images.githubusercontent.com/31124698/150302902-e2de9b80-0a98-4060-b282-bdfe2b2c1d2e.png)

Reply:
Fixed: 
![image](https://user-images.githubusercontent.com/13167207/150305785-6e1b4cc3-1fe6-4f70-ad6e-3215e6ea8d10.png)